src/test/scripts/nfs/nfs-server.yaml.template

```diff
@@ -51,12 +51,25 @@ spec:
     - name: nfs-server-container
       image: ubuntu:latest
       securityContext:
-        privileged: true
+        capabilities:
+          add:
+            - SYS_ADMIN
+            - SYS_MODULE
       command: ["/bin/sh"]
-      args: ["-c", "apt -qq update && DEBIAN_FRONTEND=noninteractive apt install -qq -y nfs-kernel-server runit inotify-tools; mkdir -p /etc/sv/nfs ; runsvdir /etc/sv"]
+      args: ["-c", "apt -qq update && DEBIAN_FRONTEND=noninteractive apt install -qq -y nfs-kernel-server runit inotify-tools kmod; mkdir -p /etc/sv/nfs ; runsvdir /etc/sv"]
       volumeMounts:
         - mountPath: /srv/nfs
           name: nfs-export-dir
+        - mountPath: /lib/modules
+          name: lib-modules
+          readOnly: true
+      env:
+        - name: NFS_EXPORT_0
```

Comment:
How is this used?

Reply:
Oops, that was a leftover from another container I used, previously. I'll remove it.